integration-tests/src/test/java/org/zowe/apiml/gatewayservice/AuthenticationOnDeploymentTest.java

```diff
@@ -146,10 +146,11 @@ public void testMultipleAuthenticationSchemes() throws Exception {
     void testReregistration() throws Exception {
 
         try (
-            final VirtualService service1 = new VirtualService("testService1", 5678);
-            final VirtualService service2 = new VirtualService("testService1", 5679);
-            final VirtualService service4 = new VirtualService("testService1", 5678)
+            final VirtualService service1 = new VirtualService("testService3", 5678);
```

Comment:
Is there any need to change the ids?

Reply:
just sometimes cant register on my local env and failed the test, rename often helps